docs/code-of-conduct.md

```diff
@@ -0,0 +1,89 @@
+# Zulip Code of Conduct
+
+## Our Commitment
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers are committed to making participation in our
+project and our community a harassment-free experience for everyone, regardless
+of age, body size, disability, ethnicity, gender identity and expression, level
+of experience, nationality, personal appearance, race, religion, or sexual
+identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being friendly and patient
+* Being respectful of differing viewpoints and experiences, and trying to
+  understand other people's points of view
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+* Assuming good faith on the part of other community members
+* Respecting the time and attention of others, in particular by doing research
+  in FAQs and with search engines before asking questions of maintainers
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Criticism of contributions which is not constructive, which occurs outside
+  project channels or which is repetitive
+* Publishing others' private information, such as a physical or electronic
+  address without explicit permission
+
+We will exclude you from interaction if you insult, demean or harass anyone.
+That is not welcome behavior.
+
+## Responsibilities of Project Maintainers
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, or to ban temporarily or permanently any
+contributor for other behaviors that they deem inappropriate, threatening,
+offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+project spaces include mailing lists, chat.zulip.org, GitHub and in code.
+Examples of representing a project or community include using an official
+project e-mail address, posting via an official social media account, or acting
+as an appointed representative at an online or offline event.
+
+## Enforcement
+
+If you feel you have been or are being harassed or made uncomfortable by a
```

Comment:
I would simplify this say say "To report code of conduct violations or concerns..."

In particular I would avoid any mention of being "made uncomfortable" since comfort is not really a meaningful distinction in terms of acceptable community behavior.